<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypingPractice</class>
 <widget class="QDialog" name="TypingPractice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="input_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:0.1 rgba(255, 255, 255, 255), stop:0.2 rgba(255, 176, 176, 167), stop:0.3 rgba(255, 151, 151, 92), stop:0.4 rgba(255, 125, 125, 51), stop:0.5 rgba(255, 76, 76, 205), stop:0.52 rgba(255, 76, 76, 205), stop:0.6 rgba(255, 180, 180, 84), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speed_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>95</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 233, 58);</string>
   </property>
   <property name="text">
    <string>타수 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="wrong_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>125</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 233, 58);</string>
   </property>
   <property name="text">
    <string>오타 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="pix_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>35</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(177, 137, 116);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../사진/스크린샷, 2022-07-02 13-12-54.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="board_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 60, 52);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../사진/스크린샷, 2022-07-02 13-31-17.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="boardback_label">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>55</y>
     <width>460</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:7px solid rgb(177, 137, 116);
border-radius: 10px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="boardback_label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>25</y>
     <width>150</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:10px solid rgb(177, 137, 116);
border-radius: 10px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="head_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>15</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../../사진/스크린샷, 2022-07-02 16-19-59.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="head_label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>15</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>짧 은 글   연 습</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="boardback_label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>170</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:7px solid rgb(177, 137, 116);
border-radius: 10px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background_label</zorder>
  <zorder>boardback_label</zorder>
  <zorder>board_label</zorder>
  <zorder>input_text</zorder>
  <zorder>exit_btn</zorder>
  <zorder>show_label</zorder>
  <zorder>speed_label</zorder>
  <zorder>wrong_label</zorder>
  <zorder>pix_label</zorder>
  <zorder>boardback_label_2</zorder>
  <zorder>head_label</zorder>
  <zorder>head_label_2</zorder>
  <zorder>boardback_label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
